src/types/stan.ts

```diff
@@ -339,6 +339,17 @@ export function createSessionStorageForLabwareAwaiting( labware:LabwareFieldsFra
   );
 }
 
+export function getCurrentDateTime() {
```

Comment:
To my knowledge, there is not one, in the way the input field field can understand (format 'YYYY-MM-DDT00:00') and display. I did some search and most are pointing to a similar solution.  The other option I think is to use a 'Time' field type . But as the graphql is expecting a string, I followed this way so that I can use the same data structure